src/utils.ts

```diff
@@ -1,3 +1,9 @@
 export function capitalizeFirstLetter(string: string) {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
+
+export function getChainColor(color: string) {
```

Comment:
I think better to call it something like `getBackgroundStyle` because here we don't receive color but style